packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -134,17 +186,34 @@ the `emitNextQuerySelected` function when the next query is selected. In the exa
 of using the default `button` tag for a next query, an icon is added, and the text of the next query
 is wrapped in a `span`
 
-```vue
-<NextQueries>
-  <template #suggestion="{suggestion}">
-    <NextQuery :suggestion="suggestion" class="x-next-queries__suggestion">
-      <template #default="{suggestion}">
-        <img src="./next-query-icon.svg" class="x-next-query__icon"/>
-        <span class="x-next-query__query">{{ suggestion.query }}</span>
-      </template>
-    </NextQuery>
-    <button>Custom Behaviour</button>
-  </template>
-</NextQueries>
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work